.circleci/config.yml

```diff
@@ -1023,6 +1060,44 @@ jobs:
       - store_test_results:
           path: test/test-results/e2e
 
+  test-e2e-mmi-playwright - OPTIONAL:
+    executor: playwright
+    parallelism: 2
+    steps:
+      - run: *shallow-git-clone
+      - attach_workspace:
+          at: .
+      - run:
+          name: Move test build to dist
+          command: mv ./dist-test-mmi-playwright ./dist
+      - run:
+          name: Install chromium
+          command: |
+            yarn playwright install chromium
+      - run:
+          name: test:e2e:chrome:mmi
+          command: |
+            SHARD="$((${CIRCLE_NODE_INDEX}+1))"; xvfb-run yarn playwright test --shard=${SHARD}/${CIRCLE_NODE_TOTAL}
+          no_output_timeout: 20m
+      - slack/notify:
+          branch_pattern: Version-v*
+          event: fail
+          mentions: <@antonio.regadas>, <@ramon.acitores134>
+          template: basic_fail_1
+          channel: C01LUJL3T98
+      - slack/notify:
+          branch_pattern: develop
+          event: fail
+          mentions: <@antonio.regadas>, <@ramon.acitores134>
+          template: basic_fail_1
+          channel: C05QXJA7NP8
```

Comment:
Develop tests run **very frequently**... are we sure this won't become noise v signal?

Reply:
As the job is optional, we need to know when the mmi build has started failing on develop (to identify cases where non mmi changes impacts the mmi build). Therefore we need this notification. Notice that will be only notified in case of failure of this job, so hopefully won't be so frequent, and MMI team should make sure it is not failing for long time (so mmi team should fix the issue asap)